tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Users/Permissions/UserGroup/DocumentPropertyValuePermission.spec.ts

```diff
@@ -0,0 +1,128 @@
+
+const testUser = ConstantHelper.testUserCredentials;
+let testUserCookieAndToken = {cookie: "", accessToken: "", refreshToken: ""};
+
+const userGroupName = 'TestPropertyValuePermission';
+let userGroupId = null;
+
+const documentName = 'TestDocument';
+const documentTypeName = 'TestDocumentType';
+const dataTypeName = 'Textstring';
+const textString = 'This is test textstring';
+
+test.beforeEach(async ({umbracoApi}) => {
+  await umbracoApi.documentType.ensureNameNotExists(documentTypeName);
+  await umbracoApi.document.ensureNameNotExists(documentName);
+  const dataTypeData = await umbracoApi.dataType.getByName(dataTypeName);
+  const documentTypeId = await umbracoApi.documentType.createDocumentTypeWithPropertyEditor(documentTypeName, dataTypeName, dataTypeData.id);
+  await umbracoApi.document.createDocumentWithTextContent(documentName, documentTypeId, textString, dataTypeName);
+});
+
+test.afterEach(async ({umbracoApi}) => {
+  // Ensure we are logged in to admin
+  await umbracoApi.loginToAdminUser(testUserCookieAndToken.cookie, testUserCookieAndToken.accessToken, testUserCookieAndToken.refreshToken);
+  await umbracoApi.documentType.ensureNameNotExists(documentTypeName);
+  await umbracoApi.document.ensureNameNotExists(documentName);
+  await umbracoApi.userGroup.ensureNameNotExists(userGroupName);
+});
+
+test('cannot see property values without UI read permission', async ({umbracoApi, umbracoUi}) => {
+  // Arrange
+  userGroupId = await umbracoApi.userGroup.createUserGroupWithReadPermissionAndReadPropertyValuePermission(userGroupName, true, false);
+  await umbracoApi.user.setUserPermissions(testUser.name, testUser.email, testUser.password, userGroupId);
+  testUserCookieAndToken = await umbracoApi.user.loginToUser(testUser.name, testUser.email, testUser.password);
+  await umbracoUi.goToBackOffice();
+
+  // Act
+  await umbracoUi.content.goToSection(ConstantHelper.sections.content, false);
+  await umbracoUi.content.goToContentWithName(documentName);
+
+  // Assert
+  await umbracoUi.content.isPropertyEditorUiWithNameVisible('text-box', false);
+});
+
+test('can see property values with UI read but not UI write permission', async ({umbracoApi, umbracoUi}) => {
+  // Arrange
+  userGroupId = await umbracoApi.userGroup.createUserGroupWithReadPermissionAndReadPropertyValuePermission(userGroupName, true, true);
+  await umbracoApi.user.setUserPermissions(testUser.name, testUser.email, testUser.password, userGroupId);
+  testUserCookieAndToken = await umbracoApi.user.loginToUser(testUser.name, testUser.email, testUser.password);
+  await umbracoUi.goToBackOffice();
+
+  // Act
+  await umbracoUi.content.goToSection(ConstantHelper.sections.content, false);
+  await umbracoUi.content.goToContentWithName(documentName);
+
+  // Assert
+  await umbracoUi.content.isPropertyEditorUiWithNameVisible('text-box');
+  await umbracoUi.content.isPropertyEditorUiWithNameReadOnly('text-box');
```

Comment:
```suggestion
  await umbracoUi.content.isPropertyEditorUiWithNameReadOnly('text-box');
```
Since it is readonly, it should be visible